ci-operator/step-registry/upi/gcp/rhel/pre/upi-gcp-rhel-pre-ref.yaml

```diff
@@ -0,0 +1,32 @@
+ref:
+  as: upi-gcp-rhel-pre
+  from: libvirt-installer
+  grace_period: 10m
+  commands: upi-gcp-rhel-pre-commands.sh
+  resources:
+    requests:
+      cpu: 10m
+      memory: 100Mi
+  env:
+    - name: HOME
+      default: /tmp/secret
+    - name: NSS_WRAPPER_PASSWD
+      default: /tmp/secret/passwd
+    - name: NSS_WRAPPER_GROUP
+      default: /tmp/secret/group
+    - name: NSS_USERNAME
+      default: rhel8user
+    - name: NSS_GROUPNAME
+      default: rhel8user
+    - name: GOOGLE_COMPUTE_IMAGE_FAMILY
+      default: rhel-8
+    - name: GOOGLE_COMPUTE_IMAGE_NAME
+      default: ''
+    - name: GOOGLE_COMPUTE_IMAGE_PROJECT
+      default: rhel-cloud
+    - name: GOOGLE_COMPUTE_MACHINE_TYPE
+      default: e2-standard-4
+    - name: GOOGLE_COMPUTE_ZONE
+      default: us-central1-d
```

Comment:
https://cloud.google.com/compute/docs/regions-zones doesn't list `us-central1-d` - maybe it was shut down because there are `us-central1-{a,b,c,e}`

Related to:
```
+ gcloud compute instances create ci-op-s2pt6lyg-1e4b3 '--image-family=rhel-8}' --image-project=rhel-cloud --zone us-central1-d --machine-type e2-standard-4 --boot-disk-type pd-ssd --subnet ci-op-s2pt6lyg-1e4b3 --network ci-op-s2pt6lyg-1e4b3 --hostname release-ci-ci-op-s2pt6lyg-1e4b3.openshift-ci.com
ERROR: (gcloud.compute.instances.create) Could not fetch resource:
 - Permission denied on 'locations/us-central1-d' (or it may not exist).
```